packages/terra-clinical-label-value-view/tests/nightwatch/TextValueLabelValueView.jsx

```diff
@@ -2,4 +2,15 @@ import React from 'react';
 
 import LabelValueView from '../../lib/LabelValueView';
 
-export default () => <LabelValueView id="LabelValueView" label="Label" textValue="Sample Text" />;
+const examples = () => (
+  <div>
+    <h3>Default Example:</h3>
+    <LabelValueView id="LabelValueView" label="Label" textValue="Sample Text" />
+    <h3>Applied max-width of 350px to show word wrapping:</h3>
+    <div style={{ maxWidth: '350px' }}>
+      <LabelValueView id="LongLabelValueView" label="A.really.long.label.value.view.sample.text.to.see.it.wrap.correctly." textValue="A.really.long.label.value.view.sample.text.to.see.it.wrap.correctly." />
```

Comment:
It's not often you get an opportunity to use `supercalifragilisticexpialidocious`